test/shared/src/main/scala/zio/test/AbstractRunnableSpec.scala

```diff
@@ -21,6 +21,7 @@ import zio._
 import zio.stacktracer.TracingImplicits.disableAutoTrace
 
 @EnableReflectiveInstantiation
+@deprecated("Use zio.test.ZIOSpecAbstract instead", "2.0.0-RC4")
```

Comment:
Would do these as `deprecated("use ZIOSpecAbstract", "2.0.0")` for consistency with other deprecation messages.